src/visuals/sprite.rs

```diff
@@ -25,6 +27,20 @@ struct EventCollection {
     additive_: Vec<Additive>,
 }
 
+pub enum LoopType {
```

Comment:
Add a small comment about it

You can view how that renders using `$ cargo doc --open`